tracee/consts.go

```diff
@@ -214,365 +204,726 @@ type EventConfig struct {
 	EssentialEvent   bool
 }
 
+// events should match defined values in ebpf code
+const (
```

Comment:
This is a conceptual duplication of `EventsNameToID`. we can eliminate one of them

Reply:
Right. Will do in a future commit